libs/libuv/Makefile

```
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libuv
-PKG_VERSION:=1.45.0
+PKG_VERSION:=1.48.0
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://dist.libuv.org/dist/v$(PKG_VERSION)/
```

```suggestion
PKG_SOURCE_URL:=https://dist.libuv.org/dist/v$(PKG_VERSION)/
```

You could probably update this at the same time, it is easily checked by opening https://dist.libuv.org/dist/ in your browser